app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        result = ""
```

Comment:
why do you need this line?

Reply:
You are right. This line is left over from an old idea, I didn't notice. Fixed